foreman/foreman.spec

```diff
@@ -424,6 +436,8 @@ cp config/settings.yaml.example config/settings.yaml
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
 %{scl_rake} assets:precompile:all RAILS_ENV=production --trace
+%{scl_rake} db:migrate RAILS_ENV=production --trace
```

Comment:
@domcleal, I've added the migration to the macro. While building discovery the migration finish successfully, cache is generated, but the build fails due to some other error. Investigating...
```
 error: File not found: /builddir/build/BUILDROOT/ruby193-rubygem-foreman_discovery-2.0.0-1.el7.centos.x86_64/opt/rh/ruby193/root/usr/share/gems/gems/foreman_discovery-2.0.0/public
```